docs/configuration.md

```diff
@@ -80,8 +80,104 @@ The NGINX Plus Router supports the following annotations:
 * `nginx.router.openshift.io/websocket`. Enables Websocket. The default is `false`.
 * `nginx.router.openshift.io/grpc`. Enables gRPC. The default is `false`.
 
+Additional annotations are available in the [TCP/UDP Load Balancing Extension](#tcpudp-load-balancing-extension).
+
 ## Configuring Edge Termination
 
-If edge termination is enabled for a host in one route, it will be enabled in all other routes that reference that host as well. Thus, you only need to configure edge termination in one route. This behavior is different from the default Router behavior, where you would have to enable edge termination in every route. 
+If edge termination is enabled for a host in one route, it will be enabled in all other routes that reference that host as well. Thus, you only need to configure edge termination in one route. This behavior is different from the default Router behavior, where you would have to enable edge termination in every route.
 
 If you still want to configure edge termination in every route that reference a particular host, please note that for all those routes the NGINX Plus Router will use edge termination settings, such as termination policy and the certificate, from a route with a name that is alphabetically last among those routes.
+
+## TCP/UDP Load Balancing Extension
+
+This section describes the configuration, features, and limitations of the TCP/UDP load balancing extension. The extension is specific to the NGINX Plus Router and not supported by other routers. For a demonstration, see the [TCP/UDP Load Balancing](../examples/tcp-udp) example.
+
+### Configuration
+
+The NGINX Plus router supports TCP/UDP load balancing through the following annotations:
+* `nginx.router.openshift.io/protocol`. Enables TCP or UDP load balancing. The accepted values are `tcp` or `udp`.
+* `nginx.router.openshift.io/port`. Specifies the port for TCP and UDP load balancing for NGINX Plus to listen on. Notes:
+    *  The specified port is not checked against the ports specified in other routes. As a result, port conflicts between routes may occur. Make sure to use unique ports per route.
+    *  You can use the same port in two routes, but only if one of them is a TCP route and the other one is a UDP route.
```

Comment:
please add 3rd node:

Make sure to open the specified port in the firewall for the protocol specified in the `nginx.router.openshift.io/protocol` annotation on every node where the Router is running. You can use iptables or firewall-cmd. For example, to open port 5353 for TCP traffic, run:
```
$ sudo iptables -I OS_FIREWALL_ALLOW -p tcp --dport 5353 -j ACCEPT
```